pkg/version/version.go

```diff
@@ -20,16 +20,24 @@ func queryPackageVersion(cmdArg ...string) string {
 		cmd := exec.Command(cmdArg[0], cmdArg[1:]...)
 		if outp, err := cmd.Output(); err == nil {
 			output = string(outp)
-			deb := false
 			if cmdArg[0] == "/usr/bin/dlocate" {
 				// can return multiple matches
 				l := strings.Split(output, "\n")
 				output = l[0]
-				deb = true
+				r := strings.Split(output, ": ")
+				regexpFormat := `^..\s` + r[0] + `\s`
+				cmd = exec.Command(cmdArg[0], "-P", regexpFormat, "-l")
+				cmd.Env = []string{"COLUMNS=160"} // show entire value
+				// dlocate always returns exit code 1 for list command
+				if outp, _ = cmd.Output(); len(outp) > 0 {
+					lines := strings.Split(string(outp), "\n")
+					line := lines[len(lines)-2] // trailing newline
```

Comment:
All errors are currently reported the same way, as `UnknownPackage`

Reply:
See what you mean, added a check for a bogus result without newlines.

Previously it could result in a crash, since it assumed a trailing newline...